test/compression.ts

```diff
@@ -1,4 +1,5 @@
 import compress from "../src/compress";
+import { cleanupFixture, setupFixture } from "./testsetup";
```

Comment:
do we need this?

Reply:
Not in this suite: i'll remove this

---
In reply to: [136672535](https://github.com/Microsoft/clarity-js/pull/28#discussion_r136672535) [](ancestors = 136672535)